brainrender/__init__.py

```diff
@@ -24,7 +24,7 @@
     # package is not installed
     pass
 
-base_dir = Path(os.path.join(os.path.expanduser("~"), ".brainrender"))
+base_dir = Path.home() / ".brainglobe" / "brainrender"
 base_dir.mkdir(exist_ok=True)
```

Comment:
```suggestion
base_dir.mkdir(parents=True, exist_ok=True)
```
We need `parents=True` to also create the `.brainglobe` folder now, in case it doesn't exist yet (e.g. on CI)